device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/convention/DeviceCommandCallback.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.azure.sdk.iot.device.convention;
+
+import com.microsoft.azure.sdk.iot.deps.convention.PayloadConvention;
+import com.microsoft.azure.sdk.iot.device.DeviceTwin.DeviceMethodData;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+
+public abstract class DeviceCommandCallback
+{
+    @Getter(AccessLevel.PACKAGE)
```

Comment:
These are set by the DeviceCommand. I don't think we should expose it to the customer.